\hypertarget{classauth__error}{}\doxysection{Класс auth\+\_\+error}
\label{classauth__error}\index{auth\_error@{auth\_error}}


Класс отлова исключений, связанных с аутентификацией  




{\ttfamily \#include $<$serv\+\_\+error.\+h$>$}



Граф наследования\+:auth\+\_\+error\+:\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=184pt]{classauth__error__inherit__graph}
\end{center}
\end{figure}


Граф связей класса auth\+\_\+error\+:\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=184pt]{classauth__error__coll__graph}
\end{center}
\end{figure}
\doxysubsection*{Открытые члены}
\begin{DoxyCompactItemize}
\item 
\mbox{\Hypertarget{classauth__error_ac3014354cfc708af39561424656a789a}\label{classauth__error_ac3014354cfc708af39561424656a789a}} 
{\bfseries auth\+\_\+error} (const std\+::string \&s)
\item 
\mbox{\Hypertarget{classauth__error_a4be1572dd7b37d1b50e9b535611d9e3c}\label{classauth__error_a4be1572dd7b37d1b50e9b535611d9e3c}} 
{\bfseries auth\+\_\+error} (const char $\ast$s)
\end{DoxyCompactItemize}


\doxysubsection{Подробное описание}
Класс отлова исключений, связанных с аутентификацией 

Отлавливает исключения, которые возбуждаются в результате ошибок функции аутентификации 

Объявления и описания членов класса находятся в файле\+:\begin{DoxyCompactItemize}
\item 
\mbox{\hyperlink{serv__error_8h}{serv\+\_\+error.\+h}}\end{DoxyCompactItemize}
